<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>Publications</title>

<script src="site_libs/header-attrs-2.29/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/flatly.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/default.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>






<link rel="stylesheet" href="style.css" type="text/css" />



<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Proctor Lab</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="projects.html">Projects</a>
</li>
<li>
  <a href="people.html">People</a>
</li>
<li>
  <a href="publications.html">Publications</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Publications</h1>

</div>


<p><img src="papers.JPG" alt="Picture of publications" width="800" 
     style="margin-left: 20px; border-radius: 8px; height: 380px; object-fit: contain; display: block; margin-bottom: 20px;"></p>
<h2>
2025
</h2>
<ul>
<li>
<strong>Proctor DM</strong>, Abraham R, Righi SE. From novice to expert:
preparing your peer review. <em>mBio</em>. 2025 Jul 9;16(7):e0043025.
<a href="https://doi.org/10.1128/mbio.00430-25" target="_blank">doi:
10.1128/mbio.00430-25</a>. Epub 2025 Jun 5.
</li>
<li>
<strong>Proctor DM</strong>, Sansom SE, Deming C, Conlan S, Blaustein
RA, Atkins TK; NISC Comparative Sequencing Program; Dangana T, Fukuda C,
Thotapalli L, Kong HH, Lin MY, Hayden MK, Segre JA. Clonal Candida auris
and ESKAPE pathogens on the skin of residents of nursing homes.
<em>Nature</em>. 2025 Mar;639(8056):1016-1023.
<a href="https://doi.org/10.1038/s41586-025-08608-9" target="_blank">doi:
10.1038/s41586-025-08608-9</a>. Epub 2025 Feb 26.
</li>
</ul>
<h2>
Pre-2025
</h2>
<ul>
<li>
Ryu EP, Gautam Y, <strong>Proctor DM</strong>, Bhandari D, Tandukar S,
Gupta M, Gautam GP, Relman DA, Shibl AA, Sherchand JB, Jha AR, Davenport
ER. Correction: Nepali oral microbiomes reflect a gradient of lifestyles
from traditional to industrialized. <em>Microbiome</em>. 2024 Dec
3;12(1):252.
<a href="https://doi.org/10.1186/s40168-024-01988-6" target="_blank">doi:
10.1186/s40168-024-01988-6</a>.
</li>
<li>
<strong>Proctor DM</strong>, Drummond RA, Lionakis MS, Segre JA. One
population, multiple lifestyles: Commensalism and pathogenesis in the
human mycobiome. <em>Cell Host Microbe</em>. 2023 Apr 12;31(4):539-553.
<a href="https://doi.org/10.1016/j.chom.2023.02.010" target="_blank">doi:
10.1016/j.chom.2023.02.010</a>.
</li>
<li>
<strong>Proctor DM</strong>, Dada N, Serquiña A, Willett JLE. Problems
with Peer Review Shine a Light on Gaps in Scientific Training.
<em>mBio</em>. 2023 Jun 27;14(3):e0318322.
<a href="https://doi.org/10.1128/mbio.03183-22" target="_blank">doi:
10.1128/mbio.03183-22</a>. Epub 2023 Apr 13.
</li>
<li>
<strong>Proctor DM</strong>. Nothing about Us without Us: the Roles of
Diverse Stakeholders in Scientific Publishing. <em>mBio</em>. 2023 Apr
25;14(2):e0268522.
<a href="https://doi.org/10.1128/mbio.02685-22" target="_blank">doi:
10.1128/mbio.02685-22</a>. Epub 2023 Mar 7.
</li>
<li>
Liang H, Jo JH, Zhang Z, MacGibeny MA, Han J, <strong>Proctor
DM</strong>, Taylor ME, Che Y, Juneau P, Apolo AB, McCulloch JA, Davar
D, Zarour HM, Dzutsev AK, Brownell I, Trinchieri G, Gulley JL, Kong HH.
Predicting cancer immunotherapy response from gut microbiomes using
machine learning models. <em>Oncotarget</em>. 2022 Jul 19;13:876-889.
<a href="https://doi.org/10.18632/oncotarget.28252" target="_blank">doi:
10.18632/oncotarget.28252</a>.
</li>
<li>
Sim CK, Kashaf SS, Stacy A, <strong>Proctor DM</strong>, Almeida A,
Bouladoux N, Chen M; NISC Comparative Sequencing Program; Finn RD,
Belkaid Y, Conlan S, Segre JA. A mouse model of occult intestinal
colonization demonstrating antibiotic-induced outgrowth of
carbapenem-resistant Enterobacteriaceae. <em>Microbiome</em>. 2022 Mar
10;10(1):43.
<a href="https://doi.org/10.1186/s40168-021-01207-6" target="_blank">doi:
10.1186/s40168-021-01207-6</a>.
</li>
<li>
Saheb Kashaf S, <strong>Proctor DM</strong>, Deming C, Saary P, Hölzer
M; NISC Comparative Sequencing Program; Taylor ME, Kong HH, Segre JA,
Almeida A, Finn RD. Integrating cultivation and metagenomics for a
multi-kingdom view of skin microbiome diversity and functions. <em>Nat
Microbiol</em>. 2022 Jan;7(1):169-179.
<a href="https://doi.org/10.1038/s41564-021-01011-w" target="_blank">doi:
10.1038/s41564-021-01011-w</a>. Epub 2021 Dec 24.
</li>
<li>
Lisco A, Hsu AP, Dimitrova D, <strong>Proctor DM</strong>, Mace EM, Ye
P, Anderson MV, Hicks SN, Grivas C, Hammoud DA, Manion M, Starrett GJ,
Farrel A, Dobbs K, Brownell I, Buck C, Notarangelo LD, Orange JS,
Leonard WJ, Orestes MI, Peters AT, Kanakry JA, Segre JA, Kong HH, Sereti
I. Treatment of Relapsing HPV Diseases by Restored Function of Natural
Killer Cells. <em>N Engl J Med</em>. 2021 Sep 2;385(10):921-929.
<a href="https://doi.org/10.1056/NEJMoa2102715" target="_blank">doi:
10.1056/NEJMoa2102715</a>.
</li>
<li>
Huang X, Welsh RM, Deming C, <strong>Proctor DM</strong>, Thomas PJ;
NISC Comparative Sequencing Program; Gussin GM, Huang SS, Kong HH, Bentz
ML, Vallabhaneni S, Chiller T, Jackson BR, Forsberg K, Conlan S,
Litvintseva AP, Segre JA. Skin Metagenomic Sequence Analysis of Early
Candida auris Outbreaks in U.S. Nursing Homes. <em>mSphere</em>. 2021
Aug 25;6(4):e0028721.
<a href="https://doi.org/10.1128/mSphere.00287-21" target="_blank">doi:
10.1128/mSphere.00287-21</a>. Epub 2021 Aug 4.
</li>
<li>
<strong>Proctor DM</strong>, Dangana T, Sexton DJ, Fukuda C, Yelin RD,
Stanley M, Bell PB, Baskaran S, Deming C, Chen Q, Conlan S, Park M; NISC
Comparative Sequencing Program; Welsh RM, Vallabhaneni S, Chiller T,
Forsberg K, Black SR, Pacilli M, Kong HH, Lin MY, Schoeny ME,
Litvintseva AP, Segre JA, Hayden MK. Integrated genomic, epidemiologic
investigation of Candida auris skin colonization in a skilled nursing
facility. <em>Nat Med</em>. 2021 Aug;27(8):1401-1409.
<a href="https://doi.org/10.1038/s41591-021-01383-w" target="_blank">doi:
10.1038/s41591-021-01383-w</a>. Epub 2021 Jun 21.
</li>
<li>
<strong>Proctor DM</strong>, Shelef KM, Gonzalez A, Davis CL, Dethlefsen
L, Burns AR, Loomer PM, Armitage GC, Ryder MI, Millman ME, Knight R,
Holmes SP, Relman DA. Microbial biogeography and ecology of the mouth
and implications for periodontal diseases. <em>Periodontol 2000</em>.
2020 Feb;82(1):26-41.
<a href="https://doi.org/10.1111/prd.12268" target="_blank">doi:
10.1111/prd.12268</a>.
</li>
<li>
Davis NM, <strong>Proctor DM</strong>, Holmes SP, Relman DA, Callahan
BJ. Simple statistical identification and removal of contaminant
sequences in marker-gene and metagenomics data. <em>Microbiome</em>.
2018 Dec 17;6(1):226.
<a href="https://doi.org/10.1186/s40168-018-0605-2" target="_blank">doi:
10.1186/s40168-018-0605-2</a>.
</li>
<li>
Goltsman DSA, Sun CL, <strong>Proctor DM</strong>, DiGiulio DB,
Robaczewska A, Thomas BC, Shaw GM, Stevenson DK, Holmes SP, Banfield JF,
Relman DA. Metagenomic analysis with strain-level resolution reveals
fine-scale variation in the human pregnancy microbiome. <em>Genome
Res</em>. 2018 Oct;28(10):1467-1480.
<a href="https://doi.org/10.1101/gr.236000.118" target="_blank">doi:
10.1101/gr.236000.118</a>. Epub 2018 Sep 19.
</li>
<li>
<strong>Proctor DM</strong>, Fukuyama JA, Loomer PM, Armitage GC, Lee
SA, Davis NM, Ryder MI, Holmes SP, Relman DA. A spatial gradient of
bacterial diversity in the human oral cavity shaped by salivary flow.
<em>Nat Commun</em>. 2018 Feb 14;9(1):681.
<a href="https://doi.org/10.1038/s41467-018-02900-1" target="_blank">doi:
10.1038/s41467-018-02900-1</a>.
</li>
<li>
<strong>Proctor DM</strong>, Relman DA. The Landscape Ecology and
Microbiota of the Human Nose, Mouth, and Throat. <em>Cell Host
Microbe</em>. 2017 Apr 12;21(4):421-432.
<a href="https://doi.org/10.1016/j.chom.2017.03.011" target="_blank">doi:
10.1016/j.chom.2017.03.011</a>.
</li>
</ul>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
